lib/xhp-bootstrap/well.php

```diff
@@ -0,0 +1,50 @@
+<?hh
+/*
+ *  Copyright (c) 2014, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+final class :bootstrap:well extends :bootstrap:base {
+  attribute
+    :div,
+    enum {
+      'default',
+      'large',
+      'small'
+    } size = 'default';
+
+  protected function render(): XHPRoot {
+    $ret = <div>
+      {$this->getChildren()}
+      </div>;
```

Comment:
- this thing can all be one line
- if not, normal is:

```Hack
$ret =
  <div>
    {$this->getChildren()}
  </div>;
```

... so that opening and closing tags line up